docs/reference/migration/migrate_1_6.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[breaking-changes-1.6]]
```

Comment:
I was wondering about that too. @clintongormley Should we record deprecations in migrate* files?